### PR TITLE
Create Cassidy Williams podcast post

### PR DESCRIPTION
**Pull request by nickmyyz**

### CODE REVIEW

**Comment by nickmyyz** [comment]
@csbailey5t Cassidy isn't the first dev advocate ever (she says as much here), so do you think it would be fair to refer to her work as pioneering? I want to say it's fair because her work is at least close to the frontier even if it's not on it

File: podcast/2.11.md [hunk 1]
```diff
@@ -0,0 +1,897 @@
...
+
+And prior to that, she held developer advocate and senior engineering positions at CodePen, Amazon, Clarify, Hacker Fund, and Venmo. And last but not least, she also has a great Twitter account, which I'm sure a lot of listeners are probably familiar with. At this point, it's very popular. She posts a lot of funny and on-point videos about developer life. Cassidy, welcome to the show.
+
+**Cassidy Williams:**
+
+Thank you so much for having me. And with the intro, I feel like such a big deal suddenly.
+
+**Beyang Liu:**
+
+You are a big deal. I think, what is it, you have like 170 thousand Twitter followers?
```
Comment (csbailey5t) [inline podcast/2.11.md:65]:
```suggestion
You are a big deal. I think, what is it, you have like 170,000 Twitter followers?
```
Not sure what the house style says on this, but the numerals seem more helpful in this situation.

File: podcast/2.11.md [hunk 2]
```diff
@@ -0,0 +1,901 @@
...
+
+And so, I think the trends will be like, yes, lots of zero config or low config type of frameworks and things but with escape hatches and highly configurable options. And one example of that that I think is going to be pretty big is the Astro framework. I've been really impressed with it so far. I don't think they've hit V1 yet–at least at the time of recording.
+
+And it just works out of the box and lets you use React. It lets you use View, it lets you use Svelte and Preact and a lot of state management solutions and stuff out of the box and it just works. And then, if you don't want it to just work, you can cut things out. You can say, "Okay. I don't want you to be able to render View. I'm not going to use it. I want my packages to be smaller. Just render React or something like that." And I really appreciate that, how their zero-client-side JavaScript is shipped to the browser unless you enable it and everything else is fully configurable.
```
Comment (csbailey5t) [inline podcast/2.11.md:879]:
```suggestion
And it just works out of the box and lets you use React. It lets you use Vue, it lets you use Svelte and Preact and a lot of state management solutions and stuff out of the box and it just works. And then, if you don't want it to just work, you can cut things out. You can say, "Okay. I don't want you to be able to render Vue. I'm not going to use it. I want my packages to be smaller. Just render React or something like that." And I really appreciate that, how their zero-client-side JavaScript is shipped to the browser unless you enable it and everything else is fully configurable.
```

File: podcast/2.11.md [hunk 3]
```diff
@@ -0,0 +1,901 @@
...
+
+**Beyang Liu:**
+
+Interesting. Very cool. Is that Astro.build, that's what it is?
+
+**Cassidy Williams:**
+
+Yes, Astro.build. And what I like about it too, and this is more on their developers than anything, is it's from the creators of Snowpack. And their next big version is actually cutting Snowpack out and it's going to be using Vite and web assembly and stuff under the hood. And I really appreciated that they were just like, "You know, we know that we like our own tools but we want to use the best tools, and sometimes we want to do something that we didn't build." And I respected that about them for being humble about that and letting ego aside and building something cool with that. So, anyway, I think Astro is the start of a bigger movement in this direction.
```
Comment (csbailey5t) [inline podcast/2.11.md:889]: I haven't checked the recording, but is it "setting ego aside"?

File: podcast/2.11.md [hunk 4]
```diff
@@ -0,0 +1,901 @@
...
+
+Preact: https://preactjs.com/
+
+11ty: https://www.11ty.dev/
+
+Slinkity: https://slinkity.dev/
+
+Snowpack: https://www.snowpack.dev/
+
+Vit: https://github.com/vitjs/vit
```
Comment (csbailey5t) [inline podcast/2.11.md:51]:
```suggestion
Vite: https://vitejs.dev/
```

**Comment by csbailey5t** [comment]
> @csbailey5t Cassidy isn't the first dev advocate ever (she says as much here), so do you think it would be fair to refer to her work as pioneering? I want to say it's fair because her work is at least close to the frontier even if it's not on it

I personally think pioneering is accurate for her, but there have been a lot of people in the space. Maybe worth trying out a few others terms, like "Defining" or ...?

**Pull request merged**